02 Writing Algorithms/30 Machine Learning/03 Popular Libraries/03 Scikit-Learn/06 Predict Labels.html

```diff
@@ -0,0 +1,14 @@
+<p>To predict the labels of new data, in the <code>OnData</code> method, get the most recent set of features and then call the <code>predict</code> method.</p>
+<div class="section-example-container">
+    <pre class="python">features, _ = self.get_features_and_labels()
+prediction = self.model.predict(new_X)
```

Comment:
Instead of making predictions for all of the samples, shouldn't we just use the most recent one?